crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/bootstrap.rs

```diff
@@ -179,16 +179,17 @@ pub async fn proxy_list(url: Url2) -> KitsuneP2pResult<Vec<Url2>> {
 
 #[cfg(test)]
 mod tests {
+    /*
```

Comment:
leftover?

Reply:
These tests were ignored previously, and now they no longer compile. Moreover, I think they are inappropriate because they are written as unit tests, but depend on the external cloudflare bootstrap service... If we were to re-write them, they should spin up a local bootstrap service mock and use that for testing.

Reply:
```suggestion
    // Leaving these tests that no longer compile in so that they may be re-written in the future.
    // Rather than directly testing against the cloudflare service, we should spin up a local rust
    // bootstrap instance to test against.
    /*
```

Reply:
thanks for explaining this. FWIW, i agree with your conclusion